hw3/js/hw3-5.js

```diff
@@ -0,0 +1,40 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const operation = prompt('Enter operation (add, sub, mult, div)', '');
+    const isAdd = operation === 'add';
+    const isSub = operation === 'sub';
+    const isMult = operation === 'mult';
+    const isDiv = operation === 'div';
```

Comment:
very nice idea with variables, but overhead. We could not have two variants at the same time(

Reply:
ok thanks, how else can i do it?